From c5f85f5c7b63a3d6a65883d675c32007f4fa2e24 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 14 Dec 2022 22:46:44 +0530
Subject: [PATCH] Disable building of hwcomposer.drm_minigbm

Build error is seen due to minigbm path change.

As hwcomposer.drm_minigbm is not used in cic_cloud, disable
building of hwcomposer.drm_minigbm.

Tracked-On: OAM-105266
Change-Id: I9e623a481a967ce0475f1e408818a64aefad5ccf
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 Android.bp | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/Android.bp b/Android.bp
index 85be0b5..d0b4f70 100644
--- a/Android.bp
+++ b/Android.bp
@@ -99,16 +99,16 @@ cc_library_shared {
     cppflags: ["-DUSE_DRM_GENERIC_IMPORTER"],
 }
 
-cc_library_shared {
-    name: "hwcomposer.drm_minigbm",
-    defaults: ["hwcomposer.drm_defaults"],
-    whole_static_libs: ["drm_hwcomposer"],
-    srcs: [
-        "platform/platformdrmgeneric.cpp",
-        "platform/platformminigbm.cpp",
-    ],
-    include_dirs: ["external/minigbm/cros_gralloc"],
-}
+// cc_library_shared {
+//    name: "hwcomposer.drm_minigbm",
+//    defaults: ["hwcomposer.drm_defaults"],
+//    whole_static_libs: ["drm_hwcomposer"],
+//    srcs: [
+//        "platform/platformdrmgeneric.cpp",
+//        "platform/platformminigbm.cpp",
+//    ],
+//    include_dirs: ["external/minigbm/cros_gralloc"],
+//}
 
 // Used by hwcomposer.drm_imagination
 filegroup {
-- 
2.39.0

